content/docs/extending-tina/overview.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Extending Tina
+id: '/docs/extending-tina/overview'
+prev: null
+next: /docs/extending-tina/validation
+---
+
+Tina has many advanced features that allow the entire CMS editing experience to be customized.
+
+## Customizing Fields
+
+Tina allows the forms to be customized thought the `ui` property. This makes experience custom for your editors. Some of the main customization features are; 
+
+- Adding [frontend validation](/docs/extending-tina/validation/) to a component so that it must be saved with the desired format
```

Comment:
```suggestion
- Adding [client-side validation](/docs/extending-tina/validation/) to a component so that a desired format can be enforced before saving
```